src/Microsoft.Win32.Registry/tests/RegistryKey/RegistryKey_OpenSubKey_str_rkpc.cs

```diff
@@ -50,7 +50,6 @@ public void NegativeTests()
         }
 
         [Fact]
```

Comment:
The test was failing on the older version of the framework but it is not failing on the latest version anymore and this is why removed the attribute. the decision for the tests is we'll always test against the latest version of the framework.

---
In reply to: [104301999](https://github.com/dotnet/corefx/pull/16706#discussion_r104301999) [](ancestors = 104301999)